test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1341,19 +1297,12 @@ class test_ucp_sockaddr_protocols : public test_ucp_sockaddr {
             reqs.push_back(sreq);
 
             if (!is_exp) {
-                rreq = do_unexp_recv(recv_buf, size, sreq, err_handling_test,
-                                     send_stop, recv_stop);
+                rreq = do_unexp_recv(recv_buf, size, sreq, send_stop,
+                                     recv_stop);
                 reqs.push_back(rreq);
             }
 
-            if (!err_handling_test) {
-                requests_wait(reqs);
-            } else {
-                /* TODO: add waiting for send request completion, when UCP requests
-                 * tracking is added */
-                sreq_release(sreq);
-                request_wait(rreq);
-            }
+            requests_wait(reqs);
```

Comment:
would this fail if we didnt track requests?
pls add comment what happens in this line..
also, where do we check for status CANCELED?

Reply:
pls add comment